db/migrate/20160523154716_create_addresses.rb

```diff
@@ -7,7 +7,7 @@ def change
       t.string :city
       t.references :state
       t.integer :zip
-      t.references :user
```

Comment:
here I would say references is actually better. they are ultimately the exact same thing but references implies a relationship within the models and improves the readability